types/lodash/tsconfig.json

```diff
@@ -17,306 +17,12 @@
         "forceConsistentCasingInFileNames": true,
         "esModuleInterop": true
     },
-    "files": [
```

Comment:
Note: change to `tsconfig.json` so all of the individual function files are included in the lint checks